Advanced DB

Filiprogrammer opened the pull request:
DB now accepts multiple values on one line seperated by a commas

Filiprogrammer committed:
Changed clock speed

Added 64 Hz clock speed
Changed default clock speed to 8 Hz

miczac commented:
Hi, I'm getting conflicts with the locally generated files:
```
$ git apply ../checkbox.patch
error: patch failed: assets/asmsimulator.min.js:1
error: assets/asmsimulator.min.js: patch does not apply
error: patch failed: index.html:19
error: index.html: patch does not apply
``` 
Those probably should be excluded from the commit.
This may also be the case with the other commits, which I didn't check so far.
Thanks!

Filiprogrammer committed:
Advanced DB

DB now accepts multiple values on one line seperated by a commas

Filiprogrammer committed:
Saving & loading

Added saving and loading

Filiprogrammer committed:
Added "Auto Assemble" checkbox

When checked automatically assembles the code when clicking the "Run" button

Filiprogrammer commented:
I just did a hard reset of the last three commits and I commited the changes without "assets/".
Does it work now?

Filiprogrammer committed:
Added memory & register editing

Registers and individual bytes of memory can now be edited using a
secondary click on the register / byte of memory

miczac commented:
Thanks for the PR! Two things:
1) The asm-code can't be saved until "Run" has been hit for the first time.

2) Q: How is the filename for saving the code generated?
    The current one looks rather arbitrary?

miczac commented:
PS: as a side effect the file's stored w/ a ".dms" extension. 
This is on Linux and macos.

miczac commented:
PLS lets talk before you change something. ;)
